File: db4on/test/src/com/db4o/test/NoInternalClasses.cs
/* Copyright (C) 2004   db4objects Inc.   http://www.db4o.com */

using System;
using j4o.lang;
using com.db4o;
namespace com.db4o.test {

   public class NoInternalClasses {
      
      public NoInternalClasses() : base() {
      }
      
      public void Store() {
         Tester.Store(new StaticClass());
      }
      
      public void Test() {
         Tester.EnsureOccurrences(new StaticClass(), 0);
      }
   }
}

File: db4oj/test/src/com/db4o/test/NoInternalClasses.java
/* Copyright (C) 2004   db4objects Inc.   http://www.db4o.com */

package com.db4o.test;

import com.db4o.*;

public class NoInternalClasses {
	
	public void store(){
		Test.store(new StaticClass());
	}
	
	public void test(){
		Test.ensureOccurrences(new StaticClass(),0);
	}

}
